tests/go.mod

```diff
@@ -3,15 +3,16 @@ module github.com/icinga/icingadb/tests
 go 1.18
 
 require (
-	github.com/go-redis/redis/v8 v8.11.4
+	github.com/go-redis/redis/v8 v8.11.5
 	github.com/go-sql-driver/mysql v1.6.0
-	github.com/goccy/go-yaml v1.9.5
+	github.com/goccy/go-yaml v1.9.6
 	github.com/google/uuid v1.3.0
 	github.com/icinga/icinga-testing v0.0.0-20220516144008-9600081b7a69
-	github.com/jmoiron/sqlx v1.3.4
-	github.com/lib/pq v1.10.5
-	github.com/stretchr/testify v1.7.0
-	go.uber.org/zap v1.21.0
+	github.com/icinga/icingadb v1.1.0
```

Comment:
It was intentional that the tests module did not import anything from the actual icingadb module but rather only tests that code by running the binary (and as a nice benefit, this avoid the question which version of icingadb should be used here). From what I can tell, it looks like you're only using `types.String` from it, so I'd suggest to just go with `sql.NullString` from the standard library instead and drop the import.